src/main/java/org/mockito/internal/junit/DefaultTestFinishedEvent.java

```diff
@@ -0,0 +1,28 @@
+package org.mockito.internal.junit;
+
+/**
+ * Created by sfaber on 11/24/16.
+ */
+public class DefaultTestFinishedEvent implements TestFinishedEvent {
```

Comment:
On that I don't mind much as it is internal, not a fan of `Impl`, but I can live with it.